elevation-plot/queries.js

```diff
@@ -7,8 +7,8 @@ import qql from 'graphql-tag';
  *   - no desired range at Berlin, Germany
  *   - EV can charge at CHadMO changers
  *   - should use climate (temperature and weather conditions)
- *   - the EV driver can drive 40 km  less than the EV specs (specs is 440 km, custom range is 400 km)
- *   - min power of chargers is 43 kWh
+ *   - the EV driver can drive 40 km less than the EV specs (specs is 440 km, custom range is 400 km)
```

Comment:
This is not correct. It would have been specified by the capacity (which can be set in km but is now set in kwh. I think we can remove it.